dev-client/src/screens/AddUserToProjectScreen/components/AddTeamMemberForm.tsx

```diff
@@ -64,7 +67,10 @@ export const AddTeamMemberForm = ({projectId}: FormProps) => {
       // Success
       else {
         const user = userOrError as UserFields;
-        navigation.navigate('ADD_USER_PROJECT_ROLE', {projectId, user});
+        const userId = user.id;
+        // Currently there is no need to access any user's preferences besides the current user
+        dispatch(addUserToCache({...user, preferences: {}}));
+        navigation.navigate('ADD_USER_PROJECT_ROLE', {projectId, userId});
```

Comment:
I think this should be in a `then` after the dispatch, since our thunks are technically all async calls?

Reply:
Resolving this, as the new changes remove the additional dispatch